src/condor_contrib/docker/Makefile

```diff
@@ -0,0 +1,102 @@
+REGISTRY:=
+DATE:=$(shell date +%Y%m%d)
+VERSION:=8.9.5
```

Comment:
It's a bit more quoting headache, but we could rename this to Makefile.in, make it a cmake configure_file, and have cmake expand the VERSION automatically.